packages/wrangler/src/publish/list.ts

```diff
@@ -0,0 +1,35 @@
+import { fetchResult } from "../cfetch";
+import { logger } from "../logger";
+
+export type ServicesListRes = [
+	{
+		id: string;
+		created_on: string;
+		default_environment: {
+			script: {
+				last_deployed_from: string;
+			};
+		};
+	}
+];
+
+export async function list(
+	accountId: string | undefined,
+	name: string | undefined
```

Comment:
This should be consistently named. Is it a prefix, name, or search term?

Reply:
The reason I used name is that's the name of the query parameter on the request. I can still change it to prefix if that would make the most sense